main.go

```diff
@@ -102,7 +102,7 @@ func (s CSVFlag) Set(v string) error {
 }
 
 func (f *Flags) parseFlags(args []string) error {
-	flags := flag.NewFlagSet("hc-bundler", flag.ExitOnError)
+	flags := flag.NewFlagSet("hcdiag", flag.ExitOnError)
```

Comment:
hello old friend, my dear "bundler"

Reply:
The last remnant of support bundler... 🖖